components/canvas/webgl_thread.rs

```diff
@@ -908,22 +903,23 @@ impl WebGLThread {
         image_key
     }
 
-    /// Updates a `ImageKey` that uses shared textures.
-    fn update_wr_external_image(
-        webrender_api: &mut RenderApi,
-        webrender_doc: DocumentId,
-        size: Size2D<i32>,
-        alpha: bool,
+    /// Tell WebRender to invalidate any cached tiles for a given `WebGLContextId`
+    /// when the underlying surface has changed e.g due to resize or buffer swap
+    fn update_wr_image_for_context(
+        &mut self,
         context_id: WebGLContextId,
-        image_key: ImageKey,
-        image_buffer_kind: ImageBufferKind,
+        size: Size2D<i32>,
+        has_alpha: bool,
     ) {
-        let descriptor = Self::image_descriptor(size, alpha);
-        let data = Self::external_image_data(context_id, image_buffer_kind);
+        let info = self.cached_context_info.get_mut(&context_id).unwrap();
```

Comment:
Hrm. Does this need to be `get_mut(...)` or can it simply be `get(...)`?

Reply:
Good catch! We can just use 'get'. Fixed in  bce77ef